index.html

```diff
@@ -48,8 +50,166 @@ <h3 class="heading-3">Let's connect</h3>
+
+    <p>
+      A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+    <ul class="languages">
+    <li class="lang" >html</li>
+    <li class="lang" >css</li>
+    <li class="lang">javascript</li>
+    </ul>
+    <button class="btn">See project</button>
+  </div>
+
+    <div class="work-section">
+    <img  class="cards" src="./images/snap-card-3.png" alt="availability">
+    <h2 class="heading-2">Multi-Post<br>Stories</h2>
+
+    <div class="specify">
+    <p class="card-text-1">Canopy</p>
+    <img  class="dot" src="./images/dot.png" alt="dot img">
+    <p class="card-text">Back End Dev</p>
+    <img class="dot" src="./images/dot.png" alt="dot img">
+    <p class="card-text">2015</p>
+    </div>
+
+    <p>
+      A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+    <ul class="languages">
+    <li class="lang" >html</li>
+    <li class="lang" >css</li>
+    <li class="lang" >javascript</li>
+    </ul>
+    <button class="btn">See project</button>
+    </div>
+
+    <div class="work-section">
+    <img class="cards" src="./images/Snap-card-5.png" alt="avocode">
+    <h2 class="heading-2">Tonic</h2>
+
+    <div class="specify">
+    <p class="card-text-1">Canopy</p>
+    <img class="dot"  src="./images/dot.png" alt="dot img">
+    <p class="card-text">Back End Dev</p>
+    <img class="dot"  src="./images/dot.png" alt="dot img">
+    <p class="card-text">2015</p>
+    </div>
+
+    <p>
+      A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+
+    <ul class="languages">
+    <li class="lang" >html</li>
+    <li class="lang" >css</li>
+    <li class="lang" >javascript</li>
+    </ul>
+    <button class="btn">See project</button>
+  </div>
+
+    <div class="work-section">
+    <img class="cards" src="./images/snap-card-2.png" alt="avocode">
+    <h2 class="heading-2">Multi-Post<br>Stories</h2>
+
+    <div class="specify">
+    <p class="card-text-1">Canopy</p>
+    <img class="dot" src="./images/dot.png" alt="dot img">
+    <p class="card-text">Back End Dev</p>
+    <img  class="dot" src="./images/dot.png" alt="dot img">
+    <p class="card-text">2015</p>
+    </div>
+
+    <p>
+      A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+    <ul class="languages">
+    <li class="lang">html</li>
+    <li class="lang" >css</li>
+    <li class="lang" >javascript</li>
+    </ul>
+    <button class="btn">See project</button>
+  </div>
+</div>
+  <!--- ===== About section ======= --->
+  <div class="about">
+    <h1 class="heading">About<br>Myself</h1>
+    <p class="paragraph">
+      I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+    </p>
+    <h3 class="heading-3">Let's connect</h3>
+    <div class="icons">
```

Comment:
- [ ] Kindly consider taking a `ul` for these icons instead of div. and then each icon would be inside an `li` and then `a` tag. this is to follow best practice and semantics.